.github/workflows/actions-pipeline.yml

```diff
@@ -0,0 +1,70 @@
+name: Build
+
+concurrency:
```

Comment:
do we need this?

Reply:
looks like that's at the workflow level so no, I think we can nix this.